src/lib/y2storage/boot_requirements_strategies/prep.rb

```diff
@@ -62,6 +58,11 @@ def warnings
 
     protected
 
+      # Maximum size firmware can handle
+      def max_prep_size
+        prep_volume.max_size_limit
+      end
+
       # PReP partition is needed, so return any warning related to it.
       def prep_warnings
         res = []
```

Comment:
So, now, if a big prep exists, the detailed warning about that (`big_prep_warning`) is not used. Why to not switch `if` and `elsif` conditions?

Reply:
Sorry, does it relate to this code block?

Reply:
This https://github.com/yast/yast-storage-ng/blob/4a77f3ce58d81d382d4b08418aef901b6b721dd7/src/lib/y2storage/boot_requirements_strategies/prep.rb#L71

I see you had to change a test due to the `if` conditions is now true when there is a big prep. As we have a specific message for that case, I would exchange `if` and `elsif` conditions.